udata_hydra/db/resource_exception.py

```diff
@@ -32,7 +32,9 @@ async def get_by_resource_id(cls, resource_id: str) -> Record | None:
             return await connection.fetchrow(q, resource_id)
 
     @classmethod
-    async def insert(cls, resource_id: str, table_indexes: dict[str, str] | None = {}) -> Record:
+    async def insert(
+        cls, resource_id: str, table_indexes: dict[str, str] | None = {}, comment: str | None = None
```

Comment:
We don't want a [mutable default value](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) in the function definition, right? I know it was there previously, should we still fix it here as the function definition is updated?

Reply:
Good point, thank you! Fixed